src/engine.h

```diff
@@ -2,7 +2,7 @@
 #ifndef ENGINE_H
 #define ENGINE_H
 
-#include <SDL2/SDL.h>
+#include <SDL.h>
```

Comment:
no deal

Reply:
my both Linux / Windows SDL has path "#include <SDL2/SDL.h>"

Reply:
? i will check the include path of visual studio :x

Reply:
i have fixed the path now it's ok !